packages/compiler/test/server/completion.ts

```diff
@@ -33,6 +33,53 @@ describe("compiler: server: completion", () => {
     ]);
   });
 
+  it("completes imports", async () => {
+    const completions = await complete(` import "┆ `, undefined, {
+      "package.json": JSON.stringify({
+        dependencies: {
```

Comment:
Can we add another test(could just be part of this one) where you also have `peerDependencies`

Reply:
committed this change